docs/workflow.rst

```diff
@@ -0,0 +1,231 @@
+********
+Workflow
+********
+
+This page provides a brief explanation of the workflow to perform APR calculations with `pAPRika`. For more detail users
+are recommended to go through the tutorials, which details further on how to setup and run APR simulations from start to
+finish.
+
+.. figure :: _static/images/flowchart.png
+   :figwidth: 500px
+   :width: 170px
+   :align: center
+
+   Flowchart of the *pAPRika* workflow for a typical APR simulation.
+
+Structure Preparation
+---------------------
+The starting structure for the APR simulation can be configured with *pAPRika*. The ``Align`` module provides functions
+to shift and orient a structure. For example, we can translate a structure to the origin and then orient the system to
+the :math:`z`-axis by running
```

Comment:
Would say something like... the APR calculation is most efficient in a rectangular box with the long axis parallel to the pulling axis. To make this easy to setup, the Align module provides...

Reply:
added